extensions/git-branch-name-generator/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Michael Bonner, Shu Ding, and Jiwon Choi
```

Comment:
Added Shu Ding credit for the [random branch generator](https://github.com/shuding/random-branch) (got permission)